.meteor/packages

```diff
@@ -9,3 +9,7 @@ iron:router
 fezvrasta:bootstrap-material-design
 nicolasmccurdy:selectize-unstyled
 jparker:gravatar
+jquery
+underscore
```

Comment:
I don't think you need to include jquery and underscore manually, they should be included with the meteor-platform package.